accounts/scwallet/wallet.go

```diff
@@ -776,16 +776,16 @@ func (w *Wallet) findAccountPath(account accounts.Account) (accounts.DerivationP
 		return nil, fmt.Errorf("scheme %s does not match wallet scheme %s", account.URL.Scheme, w.Hub.scheme)
 	}
 
-	parts := strings.SplitN(account.URL.Path, "/", 2)
-	if len(parts) != 2 {
+	url, path, found := strings.Cut(account.URL.Path, "/")
```

Comment:
This is not the same. The original code checked that there are not more that 1 occurrences of `/`, the updated code doesn't and just lumps all further occurrences together.

Reply:
I don't think that's what the original code does

> SplitN slices s into substrings separated by sep and returns a slice of the substrings between those separators.
The count determines the number of substrings to return: 

```go
parts := strings.SplitN("go/go/go", "/", 2)
fmt.Println(len(parts))
fmt.Println(parts)
// 2
// [go go/go]

parts := strings.SplitN("gogogo", "/", 2)
fmt.Println(len(parts))
fmt.Println(parts)
// 1
// [gogogo]
```

Reply:
> The original code checked that there are not more that 1 occurrences of /, 

Nope, it didn't actually